js/script.js

```diff
@@ -1,8 +1,17 @@
+function getMoveName(argMoveId){
+    if(argMoveId == 1){
+      return 'kamień';
+    } else {
+      printMessage('Nie znam ruchu o id ' + argMoveId + '.');
+      return 'nieznany ruch';
+    }
+}
+
 let randomNumber = Math.floor(Math.random() * 3 + 1);
 
 console.log('Wylosowana liczba to: ' + randomNumber);
 
-let computerMove = 'nieznany ruch';
+let computerMove = getMoveName(randomNumber);
```

Comment:
Jesli funkcja getMoveNAme bedzie poprawna to ponizszy blok `if ... else if .. else if` bedzie niepotrzebny. Foncka zwroci na nazwe ruchu dla wylosowanje dla komputera liczby. To samo pozniej dla playera bedzie mozna zrobic.